hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsSqsProperties.kt

```diff
@@ -23,12 +23,14 @@ data class HmppsSqsProperties(
     val dlqMaxReceiveCount: Int = 5,
     val visibilityTimeout: Int = 30,
     val errorVisibilityTimeout: Int = 0,
+    val propagateTracing: Boolean = false,
   )
 
   data class TopicConfig(
     val arn: String = "",
     val accessKeyId: String = "",
     val secretAccessKey: String = "",
+    val propagateTracing: Boolean = false,
   ) {
```

Comment:
Each queue and topic can have tracing turned on but currently by default it's off. We'll experiment and see if we want to default to true.